go/common/sgx/pcs/quote.go

```diff
@@ -456,6 +457,17 @@ func (qs *QuoteSignatureECDSA_P256) Verify(
 		return err
 	}
 
+	// Ensure PCK certification information matches the quote.
```

Comment:
Could these changes break consensus? Or would this just fail with a different error below?

Reply:
I think that the `qs.QESignature.Verify` below should fail unless something weird was going on. But we can skip this change for now to be extra sure.

Reply:
Let's skip these checks for now.

Reply:
Or you can only print the error.

Reply:
I would avoid pulling in the logger here. Also it seems this is actually making the unit tests fail so I would assume this check is not entirely per spec.